addons/website/models/website.py

```diff
@@ -353,8 +353,8 @@ def set_images(images):
         website.configurator_done = True
 
         # Enable tour
-        tour_asset_id = self.env['ir.asset']._get_related_assets([('name', '=', 'website.configurator_tour')])
-        tour_asset_id.sudo().write({'active': True})
+        tour_asset_id = self.env['ir.asset']._get_related_assets([('key', '=', 'website.configurator_tour')])
+        tour_asset_id.copy({'key': 'website.configurator_tour.%s' % website.id, 'website_id': website.id, 'active': True})
```

Comment:
Can you explain this code to me? Why does the key have to change per website? What does the `_get_related_assets` funtion do? Should not it just be about find the ir.asset record using the `ref` function?

Reply:
> Can you explain this code to me?

We need to make the asset website specific because we want this asset to be present and active in website created by the configurator but we don't want it to be present in website for which the configurator was not completed.

> Why does the key have to change per website?

Indeed after looking at the code of `_get_related_assets` we probably don't need distinct keys. I thought `_get_related_assets` could return multiple assets if I did not use different keys but it does seem to be the case due to the website domain added by `_get_related_assets`. I will test.

> What does the _get_related_assets funtion do?

I could probably simply do `self.with_context(active_test=False).sudo().search(domain)`

![Screenshot from 2021-06-01 16-12-19](https://user-images.githubusercontent.com/49784121/120338123-36ac4380-c2f4-11eb-96ea-2b9ce28e990a.png)

![Screenshot from 2021-06-01 16-16-24](https://user-images.githubusercontent.com/49784121/120338668-be924d80-c2f4-11eb-8178-7c00d79b43aa.png)
![Screenshot from 2021-06-01 16-16-18](https://user-images.githubusercontent.com/49784121/120338672-bf2ae400-c2f4-11eb-885f-919fbee15741.png)

Reply:
> I could probably simply do self.with_context(active_test=False).sudo().search(domain)

Or probably just using `env.ref`, which you should do if possible.

Reply:
> Indeed after looking at the code of _get_related_assets we probably don't need distinct keys.

Well actually, is not the whole point of the key to be the same so that the website specific one *overrides* the one with the same key? What you did here is probably just creating a new asset which is added *alongside* the old one (but since the old one is inactive, you don't see the difference)

cc @Arcasias (what's the point of key in the new ir.asset model exactly?)

> I could probably simply do self.with_context(active_test=False).sudo().search(domain)

Or probably just using `env.ref` like I said above ;)